Makefile

```diff
@@ -59,15 +60,33 @@ release-install: release-container
 
 verify-shell: docker-running
 	docker run -ti --rm \
-		--privileged \
 		--volume /home/cockpit:/home/user \
 		--volume $(CURDIR)/verify:/usr/local/bin \
 		--volume=/opt/verify:/build:rw \
-		--net=host --pid=host --privileged --entrypoint=/bin/bash \
+		--net=host --pid=host --entrypoint=/bin/bash \
         cockpit/infra-verify -i
 
 verify-container: docker-running
-	docker build -t cockpit/infra-verify verify
+	docker build -t cockpit/infra-verify:$(TAG) verify
+	docker tag cockpit/infra-verify:$(TAG) cockpit/infra-verify:latest
+
+verify-push: docker-running
+	{ \
+	ID=`docker images -q cockpit/infra-verify:latest`; \
+	if [ `echo "$$ID" | wc -w` -ne "1" ]; then \
+		echo "Expected exactly one image matching 'cockpit/infra-verify:latest'"; \
+		exit 1; \
+	fi; \
+	TAGS=`docker images --format "table {{.Tag}}\t{{.ID}}" | grep $$ID | awk '{print $$1}'`; \
+	if [ `echo "$$TAGS" | wc -w` -ne "2" ]; then \
+		echo "Expected exactly two tags for the image to push: latest and one other"; \
+		exit 1; \
+	fi; \
+	for TAG in $$TAGS; do \
```

Comment:
Slightly confusing to clobber the global `$TAG` variable here. (Just noticing, keep it if you like).